src/reducers/polling.js

```diff
@@ -143,18 +145,44 @@ const mockGetVoteHistory = async pollId => {
       return mockHistory1;
   }
 };
+
 // Constants ----------------------------------------------
 
 export const POLLS_REQUEST = 'polls/REQUEST';
 export const POLLS_SUCCESS = 'polls/SUCCESS';
 export const POLLS_FAILURE = 'polls/FAILURE';
 
+export const POLL_VOTE_REQUEST = 'poll/VOTE_REQUEST';
+export const POLL_VOTE_SENT = 'poll/VOTE_SENT';
+export const POLL_VOTE_SUCCESS = 'poll/VOTE_SUCCESS';
+export const POLL_VOTE_FAILURE = 'poll/VOTE_FAILURE';
+
 export const POLLS_SET_OPTION_VOTING_FOR = 'polls/SET_OPTION_VOTING_FOR';
-export const POLLS_WITHDRAW_OPTION_VOTING_FOR =
-  'polls/WITHDRAW_OPTION_VOTING_FOR';
 
 // Actions ----------------------------------------------
 
+const handleTx = ({ prefix, dispatch, txObject }) =>
```

Comment:
probably isn't necessary to have this as a separate fnc now that there's only one tx type/ prefix